conf/bootstrap.coffee

```diff
@@ -1,31 +1,30 @@
 
 module.exports =
-  '@nikitajs/core/lib/log/cli':
+  cli:
     pad: host: 20, header: 60
-  '@nikitajs/core/lib/log/md':
+  log:
     basedir: "#{__dirname}/../log"
-  '@nikitajs/core/lib/ssh/open':
-    disabled: true
+  disabled: true
+  ssh:
     host: 'XXX.XXX.XXX.XXX'
-    port: 22
+    port: 'XX'
```

Comment:
Don't introduce errors

Reply:
It was working on my side. The merging of the configuration that happens in `run.coffee` does overwrite the config accordingly. That is why I thought it was not necessary to hard-code.